### DISCUSSION TITLE
Add CFBundleURLTypes

### DISCUSSION THREAD

**Discussion by analog-vorort** [aonez/Keka#1426] [Ideas]
As Keka is our default and preferred Compression-Tool it would be a good idea if it is possible to open it straight from the company's wiki page … For this a CFBundleURLType and the necessary behaviour should be implemented. That would be great.

**Comment by aonez** [comment]
Can you define some use case in your environment?

I'm already developing some implementation of Shortcuts integration + X-Callback here: https://github.com/aonez/Keka/discussions/1417#discussioncomment-9021948

**Reply by analog-vorort** [reply]
Keka is installed on every Mac in our company. Due to certain security regulations in the company, data often has to be transmitted in encrypted form. We also use Keka for this purpose. The procedure is explained step-by-step in all how-tos (Confluence Wiki). It would be perfect if you could open the program directly by clicking on a link in the wiki.

**Reply by aonez** [reply]
Understood. The `keka://` URL scheme will be defined in the next release 🙌

**Comment by aonez** [comment]
Keka v1.3.8 is ready and contains the `keka://` URL scheme.

**Reply by analog-vorort** [reply]
You're just insane! Thank you so much!
